### PR TITLE
Document Snaps cron jobs feature

### LINKED ISSUE #1263: Add Snaps cron jobs feature page
Add a topic under **Snaps > Features** describing how to schedule cron jobs using the `onCronJob` entry point and `endowment:cronjob` permission. Link to the [cron job example package](https://github.com/MetaMask/snaps/tree/main/packages/examples/packages/cronjobs). The cron jobs card on the Snaps intro page should link to this feature page.

### PR DESCRIPTION
**Pull request by alexandratran**
# Description

Add feature page for Snaps cron jobs. Consolidate and clarify some information from the corresponding permission and entry point descriptions. Also reorder Snaps feature pages in alphabetical order.

## Issue(s) fixed

Fixes #1263

## Preview

https://docs.metamask.io/1263-cronjobs/snaps/features/cron-jobs/

## Checklist

Complete this checklist before merging your PR:

- [x] If this PR contains a major change to the documentation content, I have added an entry to the top of the ["What's new?"](https://github.com/MetaMask/metamask-docs/blob/main/docs/whats-new.md) page.
- [ ] The proposed changes have been reviewed and approved by a member of the documentation team.

### CODE REVIEW

**Review by Montoya — APPROVED** [review]

File: snaps/features/cron-jobs.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+You can schedule actions to run periodically at fixed times or intervals, also known as "cron jobs."
+For example, you can display a dialog or notification in MetaMask at a specific time each day.
+
+## Steps
+
+### 1. Configure a cron job
+
+To configure a cron job, request the [`endowment:cronjob`](../reference/permissions.md#endowmentcronjob)
+permission specifying a schedule of one or more cron jobs in the `jobs` array.
```
Comment (bgravenorst) [inline snaps/features/cron-jobs.md:16]: If a cron job is a scheduled action, is "schedule" redundant here?

```suggestion
To configure a cron job, request the [`endowment:cronjob`](../reference/permissions.md#endowmentcronjob)
permission and define one or more cron jobs in the `jobs` array.
```

File: snaps/features/cron-jobs.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Each job is defined by a [cron expression](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm)
+and request object that will be sent to the Snap's cron job handler when the job is executed.
```
Comment (bgravenorst) [inline snaps/features/cron-jobs.md:18]: This sounds a bit passive, perhaps something like:
```suggestion
Define each job with a [cron expression](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm) and a request object, which the system sends to the Snap's cron job handler upon execution.
```

File: snaps/features/cron-jobs.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      }
+    ]
+  }
+}
+```
+
+### 2. Implement a cron job handler
+
+Expose an [`onCronjob`](../reference/entry-points.md#oncronjob) entry point, which is triggered at
+the specified schedule with the specified requests defined in the `endowment:cronjob` permission.
```
Comment (bgravenorst) [inline snaps/features/cron-jobs.md:40]:
```suggestion
Expose an [`onCronjob`](../reference/entry-points.md#oncronjob) entry point, which is triggered at
the specified schedule with the requests defined in the `endowment:cronjob` permission.
```

File: snaps/features/cron-jobs.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        },
+      });
+
+    default:
+      throw new Error("Method not found.");
+  }
+};
+```
+
+:::note Access data from cron jobs
```
Comment (bgravenorst) [inline snaps/features/cron-jobs.md:67]: nit: A tip perhaps?

**Review by bgravenorst — APPROVED** [review]
Left some comment/suggestions

**Pull request merged**